tools/build_all.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
+
+# All the parenthesis around the calls to setup.bash and the build are to
+# prevent the ROS variables from entering the global scope of this script.
+# It seems that the ros1_bridge build will fail if the ros2 environment is
+# sourced before ROS 1, so we need to make sure that no ROS 2 variables are
+# set at the time we start the last step, the ros1_bridge build.
+
+set -e
+
+CWD=`pwd`
+
+# Determin which repos to build. If the ROS 1 and ROS 2 base directories are not
+# present, then don't build
+if [ -d "ros1_dependencies_ws" ]; then
+  ENABLE_ROS1=true
+else
+  ENABLE_ROS1=false
+fi
+if [ -d "ros2_ws" ]; then
+  ENABLE_ROS2=true
+  ROS2_SETUP_FILE=$CWD/ros2_ws/install/setup.sh
+else
+  ENABLE_ROS2=false
+  ROS2_SETUP_FILE=/opt/ros/bouncy/setup.sh
```

Comment:
So that this will work with Crystal, can we change bouncy to $ROS2_DISTRO and have it be an env var? That env var is already set by the Dockerfile.

Reply:
I can make path configurable based on environment variable, but that won't make it possible to just change the variable to get a different build.

The release we can use depends on the code we are building